config/default.yml

```diff
@@ -325,6 +325,14 @@ Rails/LinkToBlank:
   Enabled: true
   VersionAdded: '0.62'
 
+Rails/MatchRoute:
+  Description: >-
+                  Don’t use match to define any routes unless there is need to map multiple request types
+                  among [:get, :post, :patch, :put, :delete] to a single action using :via option.
+  StyleGuide: 'https://rails.rubystyle.guide/#no-match-routes'
+  Enabled: 'pending'
+  VersionAdded: '2.7'
```

Comment:
It seems better to only include config/routes.rb by default.

Reply:
Some large apps split their routes into multiple files, so added support for that also into `Include`.